Cabal-tests/tests/UnitTests/Distribution/Utils/Structured.hs

```diff
@@ -29,7 +29,7 @@ tests = testGroup "Distribution.Utils.Structured"
     , testCase "GenericPackageDescription" $
       md5Check (Proxy :: Proxy GenericPackageDescription) 0x8d8f340f10a58b8d8a87bf42213dac89
     , testCase "LocalBuildInfo" $
-      md5Check (Proxy :: Proxy LocalBuildInfo) 0xbb22c3258d3092f31e992bc093d09170
+      md5Check (Proxy :: Proxy LocalBuildInfo) 0x618ab257e99d0b21c617e1f8c39a5a4b
```

Comment:
Why is `LocalBuildInfo` changing?

Reply:
No idea

Reply:
Try rebasing, it should disappear

Reply:
:thinking:

Reply:
@fgaz rebasing on current 3.10 you mean?

Reply:
Nevermind, I grepped the wrong branch :see_no_evil: